src/modules/ekf2/ekf2_main.cpp

```diff
@@ -572,6 +628,13 @@ void Ekf2::run()
 {
 	bool imu_bias_reset_request = false;
 
+	// support for dual GPS inputs if available
+	int gps_subs[2];
+
+	for (unsigned i = 0; i < 2; i++) {
+		gps_subs[i] = orb_subscribe_multi(ORB_ID(vehicle_gps_position), i);
```

Comment:
Should be with _gps_subs in constructor.

Reply:
will fix